src/ui/components/Breethe/component.ts

```diff
@@ -114,7 +118,11 @@ export default class Breethe extends Component {
     this.router
       .on('/', () => this._setMode(MODE_SEARCH))
       .on('/search', () => this._setMode(MODE_SEARCH))
-      .on(/search\/(\-?\d+(\.\d+)?,\s*\-?\d+(\.\d+)?)/, (coordinates) => this._setMode(MODE_SEARCH, { coordinates }))
+      .on(/search\/(\-?\d+(\.\d+)?,\s*\-?\d+(\.\d+)?)/, (coordinatesRaw: string) => {
+        let coordinates: string[] = coordinatesRaw.match(/[-+]?[0-9]*\.?[0-9]+/g);
+        this._setMode(MODE_SEARCH, { coordinates });
+      })
```

Comment:
I think we could simply change the regexp for the route here?

```js
.on(/search\/((\-?\d+(\.\d+)?),\s*(\-?\d+(\.\d+)?))/, (_, lat, _, lon) => {
  //only have to convert lat and lon to numbers then here
}
```

Reply:
Trues! I follow this pattern now with some changes to the expression.